src/utils/index.js

```diff
@@ -120,7 +120,7 @@ export const stripNamespace = (str) => {
  * @param {String} str The string to check
  * @returns {String} The string with or without replaced values
  */
-export const prettifyName = (str) => {
+export const prettifyName = (str = '') => {
```

Comment:
The default value will only handle undefined, it will not handle cases of `null`. Though prettifyName will never be called with null, I would still suggest using an explicit check like below to handle all weird cases -
```python
export const prettifyName = (str) => {
  if (!str) {
    return ''; 
  }
  ...
```